### PR TITLE
Add CI for build and publish docker Images

### PR DESCRIPTION
**Pull request by raminqaf**

### CODE REVIEW

**Comment by philipp94831** [comment]
I guess this closes #19 as well?

File: word-count/code/build.gradle.kts [hunk 1]
```diff
@@ -1,7 +1,12 @@
 description = "Word count pipeline with Kafka Streams"
+
 plugins {
     java
     idea
+    id("net.researchgate.release") version "3.0.2"
+    id("com.bakdata.sonar") version "1.4.1"
+    id("com.bakdata.sonatype") version "1.4.1"
+    id("org.hildan.github.changelog") version "2.2.0"
```
[the remark this answers is not in the document]
Comment (raminqaf) [reply word-count/code/build.gradle.kts:9]: I don't have access to the repo 😅

Comment (philipp94831) [reply word-count/code/build.gradle.kts:9]: Whoops, wrong link https://github.com/bakdata/streams-bootstrap/blob/master/build.gradle.kts#L2

Comment (philipp94831) [reply word-count/code/build.gradle.kts]: Remove the changelog plugin

**Review by philipp94831 — APPROVED** [review]

**Pull request merged**